src/Compilers/CSharp/Portable/Symbols/Source/SourceSimpleParameterSymbol.cs

```diff
@@ -27,6 +27,19 @@ internal override ConstantValue ExplicitDefaultConstantValue
             get { return null; }
         }
 
+        internal override bool IsNullChecked
+        {
+            get
+            {
+                var node = this.GetNonNullSyntaxNode();
+                if (node is ParameterSyntax)
```

Comment:
`is ParameterSyntax param` and then you can say `return param.ExclamationToken.IsMissing`